### PR TITLE
DebDepl_v0.2

### PR DESCRIPTION
**Pull request by drexljo**
Updated the documentation on Deb installation (experimental); killed quite some errors and security issues and clarified stuff.

### CODE REVIEW

File: content/en/docs/Getting started/Installation/singleservernocontainers.md [hunk 1]
```diff
@@ -7,31 +7,30 @@ description: >
 tags: ["subtopic"]
 ---
 
-*Warning:  This deployment method is still considered an experimental method.  It is still a work in progress.  Currently, only the Okapi installation is available.  The module deployments are blocked by an unsolved issue in the configuration of each module.*
+*Warning:  This deployment method is experimental; a proof of concept maintained by a single volunteer in his spare time.  It is still work in progress and may remain so for a long time.  Currently, only an unsecured backend installation is available.*
```
Comment (marciafolio) [inline content/en/docs/Getting started/Installation/singleservernocontainers.md:10]: Are you ok with the following?

Warning:  This deployment method is still considered an experimental method.  It is still a work in progress.  Currently, installation is only available for Okapi and the backend modules with this method.  The Stripes deployment is blocked by an unsolved issue in the configuration of each module.  It is not recommended for setting up a production system at this time.

File: content/en/docs/Getting started/Installation/singleservernocontainers.md [hunk 2]
```diff
@@ -7,31 +7,30 @@ description: >
...
-* Modules and users are prefixed with 'folio-' with this method.
-* This deployment method reduces the RAM requirement of each module to about 71MB. 
+* Each module creates a new systemd service and user on the operating system, prefixed with 'folio-'
+* Each module creates a new database and role in PostgreSQL with customizable prefixes
+* No PostgreSQL superuser roles are required for Okapi or the modules.  Superuser credentials are only required during the installation.
+* Reduces the RAM requirement of each module to about 71MB. 
 
-**Note:**  This installation process is still under development and it is recommended for setting up a production system. At the moment, only the Okapi and backend modules are considered on this installer, the installation of Stripes has to be manually executed.
+**Note:**  This installation process is still under development and it is NOT recommended for setting up a production system. At the moment, only the Okapi and backend modules are considered on this installer, the installation of Stripes has to be manually executed.
```
Comment (marciafolio) [inline content/en/docs/Getting started/Installation/singleservernocontainers.md:23]: Are you ok with the following:

There are several differences between this type of deployment and the docker container deployment method.   When this type of deployment is used:

* Each module creates a new service and user on the operating system.
* Each module creates a new database and role in PostgreSQL.
* No PostgreSQL superuser roles are required for Okapi or the modules.  The superuser role is required during the installation.
* Modules and users are prefixed with ‘folio-’.
* The RAM requirement of each module is reduced to about 71MB.

File: content/en/docs/Getting started/Installation/singleservernocontainers.md (hunk 1 above)
Comment (julianladisch) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:10]: Marcia, this information is missing in your proposal:
* only a single volunteer is working on this
* he is working in his spare time only.
* therefore there is no estimated date for a complete installation
* backend installation is not secured

File: content/en/docs/Getting started/Installation/singleservernocontainers.md (hunk 2 above)
Comment (julianladisch) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:23]: Marcia, this information is missing from your proposal:
* The "new service" is a "new systemd service".
* The name of the new systemd service is prefixed with 'folio-'.
* PostgreSQL databases and roles are created with customizable prefixes.

File: content/en/docs/Getting started/Installation/singleservernocontainers.md (hunk 1 above)
Comment (marciafolio) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:10]: @julianladisch I added the point about the backend installation not being secured.  The other points are not ones that are appropriate for this type of documentation repository.  We have let users know that it is experimental, that it's still a work in progress, that there are security issues, and that it's not recommended for a production system.

Here is an updated version of the text:

Warning: This deployment method is still considered an experimental method. It is still a work in progress. Currently, installation is only available for Okapi and the backend modules with this method. The backend installation is not secured at this time. The Stripes deployment is blocked by an unsolved issue in the configuration of each module. It is not recommended for setting up a production system at this time.

File: content/en/docs/Getting started/Installation/singleservernocontainers.md (hunk 2 above)
Comment (marciafolio) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:23]: Does this work?

There are several differences between this type of deployment and the docker container deployment method. When this type of deployment is used:

- Each module creates a new systemd service and user on the operating system. The name of the new systemd service is prefixed with 'folio-'.
- Each module creates a new database and role in PostgreSQL.
- PostgreSQL databases and roles are created with customizable prefixes.
- No PostgreSQL superuser roles are required for Okapi or the modules. The superuser role is required during the installation.
- Modules and users are prefixed with ‘folio-’.
- The RAM requirement of each module is reduced to about 71MB.

**Comment by marciafolio** [comment]
@julianladisch - are you ok with the proposed changes?  Thanks.

File: content/en/docs/Getting started/Installation/singleservernocontainers.md (hunk 2 above)
Comment (julianladisch) [reply content/en/docs/Getting started/Installation/singleservernocontainers.md:23]: Yes, I agree with this proposal.

**Pull request merged**